test/test_integration.py

```diff
@@ -366,6 +416,38 @@ def test_gallery(self):
 
         self.snapshot('gallery - text input')
 
+    def test_tabs_without_children(self):
+        app = dash.Dash(__name__)
+
+        app.layout = html.Div([
+            html.H1('Dash Tabs component demo'),
+            dcc.Tabs(id="tabs", value='tab-1', children=[
```

Comment:
Let's change this to `tab-2`. I just tried this out on my machine and the first tab was selected on page load:
![image](https://user-images.githubusercontent.com/1280389/42524153-aa9436ca-843d-11e8-89ca-ca33fc20a0e5.png)
```python
app.layout = html.Div([
    dcc.Tabs(
        id="tabs",
        value='tab-2',
        children=[
            dcc.Tab(
                label='Tab 1',
                value='tab-1'
            ),
            dcc.Tab(
                label='Tab 2',
                value='tab-2'
            ),
            dcc.Tab(
                label='Tab 3',
                value='tab-3'
            )
        ]
    ),
    html.Div(id='container')
])


@app.callback(Output('container', 'children'), [Input('tabs', 'value')])
def display_content(value):
    return html.H1('This is {}'.format(value))
```

Reply:
So not sure what you meant here, but I think changing it to `tab-2` is what's breaking the tests now (or it is the mystery what's been plaguing other tests too)

Reply:
What I mean here is that if `tab-2` is the initial value, then the tab with the id `tab-2` should be selected. However, right now, this tab is not selected (`tab-1` is selected no matter what). So, we should fix this issue (setting the initial tab's value should select that particular tab) and ensure that it's fixed by using `value='tab-2'` in our test